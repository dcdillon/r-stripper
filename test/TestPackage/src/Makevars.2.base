PKG_CPPFLAGS = -Wpedantic
PKG_LIBS = -lrt
