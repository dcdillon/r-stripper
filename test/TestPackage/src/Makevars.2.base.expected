PKG_CPPFLAGS = -Wpedantic
PKG_LIBS = -lrt $(R_STRIPPER_ARGUMENT)
R_STRIPPER_ARGUMENT = -Wl,-S
